tests/Cluster.py

```diff
@@ -728,7 +729,9 @@ def bootstrap(totalNodes, prodCount, biosHost, biosPort, dontKill=False, onlyBio
                 accounts.append(initx)
 
             transId=Node.getTransId(trans)
-            biosNode.waitForTransInBlock(transId)
+            if not biosNode.waitForTransInBlock(transId):
```

Comment:
Since this is repeated all over, can we make this a reportNotFound flag to waitForTransInBlock?

Reply:
Yes, as discussed will add this to a separate PR.